modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/test/js/DatePicker/DatePicker.es.js

```diff
@@ -152,7 +152,7 @@ describe('DatePicker', () => {
 		);
 	});
 
-	xit('calls the onChange callback with a valid date and time', () => {
+	it('calls the onChange callback with a valid date and time', () => {
```

Comment:
This test is still failing when I tested locally, could you take a look please? Maybe is the case to change the test, it's behavior could be outdated.

Reply:
Ok, i will take a look at it.

Reply:
I suggest using the debug feature in the test while fixing, this way you can see whats happening in the HTML when the test is running; please see the link where it explains how to do it: https://stackoverflow.com/questions/59763739/react-testing-library-some-portion-of-debugs-output-is-not-visible

Reply:
I tested it a few times, and i don't really know what is happening, because some times it passes, and sometimes don't, even without any changes. I believe the cause is the function userEvent.type, because it is a promise, so it don't completely load before the test end, and then it breaks (the reason i believe this is happening its because the console shows that only the hour input is filled). So i'm not sure what i can do to solve this.